### ISSUE TITLE
[v0.15] Release Tracker

### ISSUE THREAD

**Issue by NicolasHug** [pytorch/vision#7283]
We cut a [release branch](https://github.com/pytorch/vision/tree/release/0.15) for the 0.15 release. (common commit  with main is a192c95e77a4a4de3a8aeee45130ddc4d2773a83)

This issue is for tracking cherry-picks to the release branch.
The POC for the TorchVision team is @nicolashug

## Cherry-Pick Criteria

Only low-risk changes may be cherry-picked from main:

1. Fixes to regressions against the most recent minor release (e.g. 0.12 for 0.13 release); 
2. Critical fixes for: [silent correctness](https://github.com/pytorch/pytorch/issues?q=is%3Aissue+is%3Aopen+label%3A%22topic%3A+correctness+%28silent%29%22), [backwards compatibility](https://github.com/pytorch/pytorch/issues?q=is%3Aissue+is%3Aopen+label%3A%22topic%3A+bc-breaking%22+), [crashes](https://github.com/pytorch/pytorch/issues?q=is%3Aissue+is%3Aopen+label%3A%22topic%3A+crash%22+), [deadlocks](https://github.com/pytorch/pytorch/issues?q=is%3Aissue+is%3Aopen+label%3A%22topic%3A+deadlock%22+), (large) [memory leaks](https://github.com/pytorch/pytorch/issues?q=is%3Aissue+is%3Aopen+label%3A%22topic%3A+memory+usage%22+)
3. Fixes to new features introduced in the most recent minor release (e.g. 0.12 for 0.13 release)
4. Test/CI fixes
5. Documentation improvements
6. Compilation fixes or ifdefs required for different versions of the compilers or third-party libraries
7. Release branch specific changes (e.g. change version identifiers)

Hud: https://hud2.pytorch.org/hud/pytorch/vision/release%2F0.15/0

**Comment by NicolasHug** [comment]
main: https://github.com/pytorch/vision/pull/7288
release branch: https://github.com/pytorch/vision/pull/7298

**Comment by NicolasHug** [comment]
https://github.com/pytorch/vision/pull/7334 cherry-picks all PRs made to `main` from https://github.com/pytorch/vision/pull/7295 to https://github.com/pytorch/vision/pull/7330 (included). They're all doc or tests / fix PRs.

**Comment by NicolasHug** [comment]
When they are merged, we should also cherry-pick https://github.com/pytorch/vision/pull/7333 and #7332

**Comment by NicolasHug** [comment]
main (actually fbcode): https://github.com/pytorch/vision/commit/2336aecaff076481cdc206ea1c71477868299f2d
release branch: https://github.com/pytorch/vision/pull/7343

**Comment by NicolasHug** [comment]
main #7332
release branch: TODO

**Comment by NicolasHug** [comment]
https://github.com/pytorch/vision/pull/7352 cherry-picks various unit-test fixes https://github.com/pytorch/vision/pull/7347 https://github.com/pytorch/vision/pull/7351 https://github.com/pytorch/vision/pull/7350 https://github.com/pytorch/vision/pull/7348

**Comment by NicolasHug** [comment]
CI-fixies:

main: https://github.com/pytorch/vision/pull/7357
release-branc: #7358

**Comment by NicolasHug** [comment]
Fix flaky tests + smoke test
main: https://github.com/pytorch/vision/pull/7365 and https://github.com/pytorch/vision/pull/7362
release branch: https://github.com/pytorch/vision/pull/7367

**Comment by NicolasHug** [comment]
Follow-up from above:

https://github.com/pytorch/vision/pull/7365 should still be cherry-picked but we need to wait for the RC2 build to be available:

> This is expected to work in PyTorch-2.0 branch, but only after RC2 build (that includes cherry-pick for https://github.com/kulinseth/pytorch/commit/4886ab1ce1aaa3956c311c67c147bfb93c0a2f89 ) becomes available

https://github.com/pytorch/vision/pull/7365#issuecomment-1450255357

**Comment by NicolasHug** [comment]
Smoke test:

main: https://github.com/pytorch/vision/pull/7359
release-branch: TODO

**Comment by NicolasHug** [comment]
CI Fix
main: https://github.com/pytorch/vision/pull/7370
release branch: https://github.com/pytorch/vision/pull/7371

**Comment by NicolasHug** [comment]
Fix error message:

main: https://github.com/pytorch/vision/pull/7394
release branch: https://github.com/pytorch/vision/pull/7395

Not sure we'll trigger a  new RC. I'm still including it in case we do because it's still nice to have.

**Comment by ddelange** [comment]
is [`torch>=2.0`](https://pypi.org/project/torch/2.0.0/) support planned for v0.15?

**Comment by NicolasHug** [comment]
Hi @ddelange , yes pytorch 2.0 is supported by torchvision 0.15. Following our install instructions should get you those latest versions with either `pip` or `conda`: https://pytorch.org/get-started/locally/

**Comment by NicolasHug** [comment]
The release is done so this tracker is obsolete now - closing!
